src/osx/cocoa/dataview.mm

```diff
@@ -2305,6 +2305,10 @@ void UpdateWithRow(int row)
 
     MaxWidthCalculator calculator(m_OutlineView, column, pos);
 
+    bool calculateAllRows = fitRowHeight || ((GetColumn(pos)->GetWidthVariable() == wxCOL_WIDTH_AUTOSIZE)
+        || (m_expanderWidth == -1 && column == [m_OutlineView outlineTableColumn]));
```

Comment:
I'd really appreciate a comment explaining what's going on here and what does this variable mean because I think I'm misunderstanding its purpose.